tests/unit_tests/libs_tests/CMakeLists.txt

```diff
@@ -1,2 +1,3 @@
 add_subdirectory(ECS_tests)
-add_subdirectory(SFML_tests)
\ No newline at end of file
+add_subdirectory(LibsClient_tests)
+#add_subdirectory(SFML_tests)
```

Comment:
why removing SFML_tests ?

Reply:
(maybe add a comment line to explain why)

Reply:
It has been removed due to the build issue, but this issue is fixed now, so merge the main branch into yours.